wp-config-sample.php

```diff
@@ -8,7 +8,7 @@
  *
  * This file contains the following configurations:
  *
- * * MySQL settings
+ * * SQL settings

```

Comment:
@audrasjb How about `Database` instead? Why? These settings are not specifically sql commands, but rather constants to define the database settings.

```suggestion
 * * Database settings
```